spec/controllers/api/v1/infos_identite_entreprise_controller_spec.rb

```diff
@@ -1,11 +1,11 @@
 require 'rails_helper'
 
-describe API::InfosIdentiteEntrepriseController, type: :controller do
+describe API::V1::InfosIdentiteEntrepriseController, type: :request do
```

Comment:
Je me suis renseigné sur `type: :request` qui en effet a l'air d'être encouragé plutôt que `type: :controller`. Je plussoie ce choix.

Le "nouveau" `get` renvoie juste le http status, et donc ça n'est pas un `subject`. 
D'ailleurs tu ne l'utilise jamais en tant que tel, en vrai ça cache un `before` d'ailleurs tu mets `subject` au début de tous les tests. 

On ne peut pas le remplacer par un `before` car on s'en sert déjà pour préparer la base de donnée avant les tests ils ne s’appellerait pas dans le bon ordre. 

Là j'ai l'impression que c'est mal utiliser la logique de RSpec, ce code cache ces deux problèmes. Faudrait trouver mieux à mon avis.
(on perd aussi les onliners que j'aime bien, mais ça je peux m'en passer ^^)

Reply:
Oui il y a du refactor à faire côté specs pour les contrôleurs. 
Le souci avec `type: :controller` c'est que Rails infère automatiquement des noms de routes invalides avec des simples `get :show` depuis l'ajout de version des APIs. J'ai cherché à garder le type `:controller` pour les specs sans succès.

Je te laisse regarder si tu veux, mais ça ira dans une autre PR. L'important c'est que lundi prochain il y ait un point d'accès en prod pour les API et qu'il ne change plus.

Pour les one-liners, à mon avis plusieurs solutions. Par exemple, par exemple créer nos propres helpers du genre `it { is_expected.to have_http_status(200) }`. On peut sûrement mettre aussi la réponse de la requête dans le *subject*.